serverlessworkflow/modules/ROOT/pages/use-cases/timeout-showcase-example.adoc

```diff
@@ -108,7 +108,7 @@ service/jobs-service-postgresql created
 deployment.apps/jobs-service-postgresql created
 trigger.eventing.knative.dev/jobs-service-postgresql-create-job-trigger created
 trigger.eventing.knative.dev/jobs-service-postgresql-cancel-job-trigger created
-sinkbinding.sources.knative.dev/jobs-service-postgresql-sb created
+= sinkbinding.sources.knative.dev/jobs-service-postgresql-sb created
```

Comment:
?

Reply:
wrong change, thanks for pointing out that